src/commands/Utility/quote.ts

```diff
@@ -131,6 +137,7 @@ async function findQuote(randomTip: string, interaction: Discord.CommandInteract
 			.setTitle(quote.quote)
 			.setDescription(`      - ${author.join(" and ")}`)
 			.setFooter(randomTip, ((interaction.member as Discord.GuildMember) ?? interaction.user).displayAvatarURL({ format: "png", dynamic: true }))
+	if (quote.attachmentURL) embed.setImage(quote.attachmentURL)
```

Comment:
Consistency with the option name? it should probably be quote.image or imageURL if you will

Reply:
Fair enough